lib/git-lfs-s3/version.rb

```diff
@@ -1,3 +1,3 @@
 module GitLfsS3
-  VERSION = '0.1.1'
+  VERSION = '0.2.2'
```

Comment:
I incremented it in 0.1.x for the public server changes. Then I bumped it to 0.2.x when I added Ceph support since there were a normal of significant code changes. See: 

https://rubygems.org/gems/lsst-git-lfs-s3

And yeah, I agree no reason to push version back. It just served my needs getting the service deployed.